llama-index-integrations/tools/llama-index-tools-cassandra/llama_index/tools/cassandra/cassandra_database_wrapper.py

```diff
@@ -0,0 +1,652 @@
+"""Apache Cassandra database wrapper."""
+from __future__ import annotations
+
+import re
+import traceback
+from typing import Any, Dict, List, Optional, Sequence, Tuple, Union
+
+from cassandra.cluster import ResultSet, Session
+from pydantic import BaseModel, Field, root_validator
```

Comment:
probably safest to import pydantic from our bridge: `from llama_index.core.bridge.pydantic import ...`

Seems like some pydantic issues in the cicd

Reply:
Changed and merged.